### PR TITLE
Document severity support

### LINKED ISSUE #97: Document severity
It would be convenient to be able to link to a severity doc from our notifier libraries.

### PR DESCRIPTION
**Pull request by kyrylo**
Fixes #97 (Document severity)

### CODE REVIEW

File: jekyll/_docs/airbrake-faq/what-is-severity.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+description: what is severity
+---
+
+### What is severity?
+
+![severity.png](/docs/assets/img/docs/airbrake/severity.png)
+
+Big Airbrake projects receive a lot of errors. Every new error triggers an email
+notification. Not every error is equally important, and thus a big amount of
+emails can be overwhelming.
```
Comment (mmcdaris) [inline jekyll/_docs/airbrake-faq/what-is-severity.md]: What do you think of?
>Not every error is equally important, and a large amount of error emails can be overwhelming.

Comment (kyrylo) [inline jekyll/_docs/airbrake-faq/what-is-severity.md]: Thanks, fixed.

File: jekyll/_docs/airbrake-faq/what-is-severity.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* `error`
+* `critical`
+* `alert`
+* `emergency`
+* `invalid`
+
+Airbrake does not send email notifications for errors with the severity of
+`debug`, `info`, `notice`, `warning`. If your account has any integrations
+configured, Airbrake won't propagate errors with the above-mentioned severities,
+too.
```
Comment (kyrylo) [inline jekyll/_docs/airbrake-faq/what-is-severity.md]: @mmcdaris PTAL at this last section. I just added it.

Comment (mmcdaris) [inline jekyll/_docs/airbrake-faq/what-is-severity.md]: What do you think of shortening it to:

> Errors with a severity of `debug`, `info`, `notice`, or `warn` will not trigger error emails or integration notifications.

This information might also deserve a place in [what-triggers-a-new-email.md](https://github.com/airbrake/airbrake-docs/blob/master/jekyll/_docs/airbrake-faq/what-triggers-a-new-email.md)

Comment (kyrylo) [inline jekyll/_docs/airbrake-faq/what-is-severity.md]: Sounds good 👍

**Comment by kyrylo** [comment]
@mmcdaris PTAL. I added a new section in `what-triggers-a-new-email.md`

**Pull request merged**